docs/imm-otel.md

```diff
@@ -0,0 +1,101 @@
+
+## Deploy an OpenTelemetry Collector for SignalFx
+
+After installing the SignalFx Smart Agent on each host, you can deploy an OpenTelemetry Collector in each datacenter/region/cluster where traced applications run. In general, an OpenTelemetry Collector should receive data from the SignalFx Smart Agent.
+
+The OpenTelemetry Collector uses pipelines to receive, process, and export trace data with components conveniently known as receivers, processors, and exporters. Set up pipelines with services. You can also add extensions that provide an OpenTelemetry Collector with additional functionality, such as diagnostics and health checks. The OpenTelemetry Collector has two versions: a [core version](https://github.com/open-telemetry/opentelemetry-collector "Core Version") and a [contributions version](https://github.com/open-telemetry/opentelemetry-collector-contrib "Contributions"). The core version provides receivers, processors, and exporters for general use. The contributions version provides receivers, processors, and exporters for specific vendors and use cases.
+
+SignalFx uses the components described in the following table to send data to an OpenTelemetry Collector and to receive data from an OpenTelemetry Collector:
+
+| **Component** | **Name**        | **Description**                                                                                       |  
+|---------------| :---------------:  |-------------------------------------------------------------------------------------------------------|
+| Receiver      | `signal-fx`     | Component that sets the endpoint for receiving metrics data with the SignalFx metric data format.     |      
+| Processor     | `memory_limiter`, `batch`, and `queued_retry` are the recommended processors.         | Component that pre-processes data before it is exported.                                              |      
+| Exporter      | `signal-fx`     | Component that forwards data to SignalFx with the metric data format.                                 |
+
+  
+See [Configuration](https://opentelemetry.io/docs/collector/configuration/ "OpenTelemetry Collector Configuration") for more information about the OpenTelemetry Collector components.
+
+To deploy an OpenTelemetry Collector:
+
+1. Download the [latest release](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases "OpenTelemetry Collector contributions releases") of the OpenTelemetry Collector contributions version from GitHub.
+2. Create a configuration file that defines components for the OpenTelemetry Collector. A sample configuration file is provided below. See [examples](https://github.com/open-telemetry/opentelemetry-collector/tree/master/examples “OTeL examples”) for an OpenTelemetry Collector demo.
+```
```

Comment:
I would strongly advise to link to the github config as it will be updated while the docs likely will not